src/providers/postgres/qgspostgresproviderconnection.cpp

```diff
@@ -255,6 +255,18 @@ QList<QVariantList> QgsPostgresProviderConnection::executeSqlPrivate( const QStr
             {
               vType = QVariant::Bool;
             }
+<<<<<<< HEAD
```

Comment:
Shouldn't these markers (lines 258, 259, and 269) of the merge conflict be removed of the file?. At least the test complains about it.

Reply:
of course, the backport bot did a bad job here.